WultraMobileTokenSDK/Operations/Utils/WMTTOTPUtils.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright 2023 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import Foundation
+
+/// Utility class used for handling TOTP
+public class WMTTOTPUtils {
+    
+    /// Method accepts deeeplink URL and returns payload data
+    public static func parseLoginDeeplink(url: URL) -> WMTOperationTOTPData? {
+        
+        guard let components = URLComponents(string: url.absoluteString) else { return nil }
+        
+        guard let host = components.host, host == "login" else { return nil }
+        
+        guard let queryItems = components.queryItems else { return nil }
+        
+        guard let code = queryItems.first?.value else { return nil }
```

Comment:
I would log some text for each guard for developer to understand why it was not parsed